app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -801,14 +797,14 @@ def prov_set_show_vars
     else
       @options = @miq_request.options
       @options[:memory], @options[:mem_typ] = reconfigure_calculations(@options[:vm_memory][0]) if @options[:vm_memory]
-      @force_no_grid_xml   = true
+      @force_no_grid_xml = true
 
       @selected_ids = @miq_request.options[:src_ids]
       @view, @pages = get_view(Vm, :view_suffix => "VmReconfigureRequest")
       # FIXME: use selected_ids passed through get_view to replace all of
       # gtl_selected_records after Gaprindashvili
     end
-    @user = User.find_by_userid(@miq_request.stamped_by)
+    @user = User.find_by(:userid => @miq_request.stamped_by)
```

Comment:
Also not sure, possibly missing `in_my_region` now..

Reply:
I reverted the above line.